src/main/java/com/chess/backend/services/ChessboardService.java

```diff
@@ -0,0 +1,58 @@
+package com.chess.backend.services;
+
+import com.chess.backend.gamemodel.*;
+import com.chess.backend.gamemodel.constants.PieceType;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ChessboardService {
+
+    public Chessboard initNewGameBoard(String[] players){
+        Chessboard chessboard = new Chessboard();
+        chessboard.setNumberOfPlayers(players.length);
+        Square[][] squares = new Square[10][4];
+
+        for(int i=0;i< players.length; i++){
+            Player player = new Player(players[i], i);
+            this.initPlayerPieces(squares, player);
+        }
+
+        chessboard.setSquares(squares);
+        return chessboard;
+    }
+
+    private Square[][] initPlayerPieces(Square[][] squares, Player player) {
+        initPlayerPawns(squares, player);
+        initPlayerPawns(squares, player);
+        return squares;
+    }
+
+    private Square[][] initPlayerPawns(Square[][] squares, Player player) {
+        int playerFirstColumn = player.getId() * 8;
+        for(int i =0; i< squares.length;i++){
+            this.setPiece(i, playerFirstColumn, squares, new Piece(PieceType.PAWN, player, false));
```

Comment:
I think it has to be true instead of false for one of the lines where a pawn is created. Because the lower pawns go clockwise (so value has to be true) and the upper pawns go counterclockwise (so the value has to be false)

Reply:
you right thanks